internal/localize/lang.go

```diff
@@ -1,26 +1,17 @@
 package localize
 
-type Lang int
+//go:generate enumerator -type Lang -linecomment -empty
+type Lang byte
 
-func (l Lang) String() string {
-	if l == Cy {
-		return welshAbbreviation
-	}
-
-	return englishAbbreviation
-}
```

Comment:
I think we were relying on this in summary pages but I'm not sure it makes sense now its an explicit choice the user has to make.